scripts/auto-rebase/rebase.sh

```diff
@@ -770,6 +771,35 @@ update_openshift_manifests() {
     #       manifests we're manually updating them as needed for now.
     # TODO: Enable in assets.yaml and handle modifications
 
+    #-- csi-snapshot-controller ---------------------------
+    local target="${REPOROOT}/assets/components/csi-snapshot-controller/csi_controller_deployment.yaml"
+    yq -i '.metadata.namespace = "kube-system"' $target
+    yq -i '.spec.template.spec.containers[0].image = "{{ .ReleaseImage.csi_snapshot_controller }}"' $target
+    yq -i '.spec.template.spec.containers[0].args = [ "--v=2", "--leader-election=false"]' $target
+    yq -i 'del(.spec.template.spec.priorityClassName) | del(.spec.template.spec.containers[0].securityContext.seccompProfile)' $target
+    yq -i 'with(.spec.template.spec.containers[0].securityContext; .runAsUser = 65534)' $target
+
+    local target="${REPOROOT}/assets/components/csi-snapshot-controller/webhook_deployment.yaml"
+    yq -i '.metadata.namespace = "kube-system"' $target
+    yq -i '.spec.template.spec.containers[0].image = "{{ .ReleaseImage.csi_snapshot_validation_webhook }}"' $target
+    yq -i 'with(.spec.template.spec.containers[0].args;  .[] |= sub("\${LOG_LEVEL}", "2") )' $target
+    yq -i 'del(.spec.template.spec.priorityClassName)' $target
+    yq -i 'with(.spec.template.spec.containers[0].securityContext; .runAsUser = 65534)' $target
```

Comment:
Same here, why that ID?

Reply:
ditto - totally arbitrary and sufficiently low-privileged